CMakeScripts/JoinPaths.cmake

```
@@ -12,10 +12,12 @@ function(join_paths joined_path first_path_segment)
     set(temp_path "${first_path_segment}")
     foreach(current_segment IN LISTS ARGN)
         if(NOT ("${current_segment}" STREQUAL ""))
-            if(IS_ABSOLUTE "${current_segment}")
-                set(temp_path "${current_segment}")
+            string(REGEX REPLACE "^${CMAKE_INSTALL_PREFIX}" "" tmp_segment "${current_segment}")
```

Maybe something like this would allow removing the second replacement:

```suggestion
            string(REGEX REPLACE "^${CMAKE_INSTALL_PREFIX}/?" "" tmp_segment "${current_segment}")
```

---

Thanks for pointing this out, I didn't notice `?` in the cmake docs before.